torch/testing/_internal/common_fsdp.py

```diff
@@ -1352,6 +1355,40 @@ def _test_fsdp_parity(
             )
 
 
+def test_compiled_fsdp(compile_compute_on_module: Optional[type] = None):
```

Comment:
We may want to generalize this to a `Sequence[Type]` later (or some other container data structure).

Reply:
you are right. will generalize this to ``Sequence`` in 2nd PR